package.json

```diff
@@ -36,6 +36,7 @@
     "karma-mocha-reporter": "^2.2.5",
     "karma-rollup-preprocessor": "^6.1.0",
     "mocha": "^5.2.0",
+    "mongodb-core": "3.1.11",
```

Comment:
I would prefer not to make `mongodb-core` a dependency of this project, in particular because the plan is to use this version of bson in that module eventually. I think a better approach might be for you to create an object that "looks" like the old `ObjectID`, or even just copy the basic parts of the old implementation into the tests.

Reply:
Yep, done. Note that the new tests look at interop for all BSON types (not just ObjectId), and also they test serialization code interop (4.x objects serialized by 1.x). So unless we want to remove those tests, the easiest solution here (what I just checked in) was to copy/paste the entire js-bson 1.x codebase into the tests folder. This will be easy to update if there are more changes to the 1.x branch-- simply install the new 1.x from npm and copy/paste.